### PR TITLE
Add proposed quorum rules

### PR DESCRIPTION
**Pull request by dthaler**
Adapted from Stephen Walli's document, except that "majority"
is changed to "super majority" for consistency with existing text in this doc.

Signed-off-by: Dave Thaler <dthaler@microsoft.com>

### CODE REVIEW

File: docs/Maintainers.md [hunk 1]
```diff
@@ -13,6 +13,15 @@ but when that fails, the Committee calls for a vote where the super majority
 (two-thirds) wins. This is to prevent obstructionism by removing the possibility
 of a one person veto.
 
+Quorum for Community Maintenance Committee meetings requires at least two-thirds
```
Comment (andschwa) [inline docs/Maintainers.md]: Depending on PR ordering, we may have to change this over to Governance.

**Comment by johnkord** [comment]
I will add this to the agenda for the next CGC meeting.

**Comment by andschwa** [comment]
We called a vote in today's CGC meeting and are merging this PR.

File: docs/Maintainers.md (hunk 1 above)
Comment (dthaler) [reply docs/Maintainers.md]: Updated name

**Pull request merged**